### PR TITLE
Fill enrollment trend with previous day's count.

### PR DESCRIPTION
**Pull request by dsjen**
@rocha @clintonb 

FYI: @lamagnifica 

Gaps:
gaps

Filled Gaps:
filled

### CODE REVIEW

File: analytics_dashboard/courses/presenters.py [hunk 1]
```diff
@@ -205,12 +205,13 @@ def _fill_trend(self, api_response):
...
                 if current_date > expected_date:
-                    api_response.insert(day_change, self._build_empty_trend(expected_date))
+                    api_response.insert(day_change, self._create_enrollment_trend(expected_date,
+                                                                                  api_response[day_change]['count']))
 
         return api_response
 
-    def _build_empty_trend(self, day):
-        trend = {'date': day.isoformat(), 'count': 0}
+    def _create_enrollment_trend(self, day, count):
```
Comment (clintonb) [inline analytics_dashboard/courses/presenters.py]: Minor: This is creating a datapoint, not a trend. `_create_enrollment_datapoint` might be more appropriate.

Comment (dsjen) [inline analytics_dashboard/courses/presenters.py]: I like that better too--updated.

**Comment by dsjen** [comment]
Would anyone like to leave additional comments?

**Comment by clintonb** [comment]
No. I thought I approved this earlier. Oops. :+1:

**Pull request merged**